packages/css-in-js/CHANGELOG.md

```
@@ -0,0 +1,224 @@
+## 0.32.0
+
+### Minor Changes
+
+- [#1933](https://github.com/RocketChat/fuselage/pull/1933) [`00052dc`](https://github.com/RocketChat/fuselage/commit/00052dc976bbe72664ec2d3b97f9ba2ba2d4f3c7) Thanks [@gabriellsh](https://github.com/gabriellsh)! - feat(css-in-js): Allow passing a target `document` to rule attacher
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+# [0.31.0](https://github.com/RocketChat/fuselage/compare/v0.30.1...v0.31.0) (2021-12-28)
+
+### Features
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)